EIPS/eip-1.md

```diff
@@ -37,11 +37,31 @@ An EIP must meet certain minimum criteria. It must be a clear and complete descr
 
 ## EIP Work Flow
 
+### Shepherding an EIP
+
 Parties involved in the process are you, the champion or *EIP author*, the [*EIP editors*](#eip-editors), and the [*Ethereum Core Developers*](https://github.com/ethereum/pm).
 
-:warning: Before you begin, vet your idea, this will save you time. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be rejected based on prior research (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used. Examples of appropriate public forums to gauge interest around your EIP include [the Ethereum subreddit], [the Issues section of this repository], and [one of the Ethereum Gitter chat rooms]. In particular, [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your EIP.
+Before you begin writing a formal EIP, you should vet your idea. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. Proposing an EIP which has not been properly dilligenced may be grounds for EIP Editors not granting the "Draft" status. It is thus recommended to open a discussion thread on [the Ethereum Magicians forum] to do this, but you can also use [one of the Ethereum Gitter chat rooms], [the Ethereum subreddit] or [the Issues section of this repository]. 
+
+In addition to making sure your idea is original, it will be your role as the champion to socialize your EIP and build community consensus around it. You should try and gauge whether the interest in your EIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. For example, the community interest required for a technically complex Core EIP will have to be much greater than for a simple ERC. While "community interest" will not be a factor in deciding whether an EIP gets merged as a draft, insufficient community interest may prevent your EIP from moving forward, especially if it is a Core EIP. 
```

Comment:
@carver I feel like this line addresses your previous comment:

> You should try and gauge whether the interest in your EIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. For example, the community interest required for a technically complex Core EIP will have to be much greater than for a simple ERC.

What do you think?

Reply:
Yeah, I think this is most of the way there. It seems to imply that the the reason for the need of community consensus is `technical complexity`. Naturally, a message signing standard doesn't require consensus, even if it's very complex. On the flip side, even very simple changes require consensus, if it requires a hard fork (at least in the form of lack of opposition). So I guess we can be even more explicit, that it is especially important to gauge and build community support toward consensus on any network upgrade EIP.

Reply:
Good point. Should I just remove the `technically complex` and `simple`? 

Re: network upgrade, I feel like the sentence a bit after addresses this:
> insufficient community interest may prevent your EIP from moving forward, especially if it is a Core EIP. 

Any way you'd suggest rephrasing that part?

Reply:
"insufficient community interest or support may prevent your EIP from moving forward, especially if it is a Core EIP".

Either a change doesn't have enough interest from community members to make the effort worth the change. In essence, a feature that takes a lot of work (due to complexity) but no one, or not enough people, are asking for it. 

This text covers this case.
> For example, the community interest required for a technically complex Core EIP will have to be much greater than for a simple ERC.

Or, a feature has a significant impact on the community, whether or not it is technically complex,  and there isn't enough support from the community to justify making the change.

This text covers this case.
> "insufficient community interest or support may prevent your EIP from moving forward, especially if it is a Core EIP".

Reply:
I think between the two of them we are covered. :thumbs-up:

Reply:
> Good point. Should I just remove the `technically complex` and `simple`?
> 

Yeah, actually, I think removing those two qualifiers is an improvement.

> Re: network upgrade, I feel like the sentence a bit after addresses this:
> 
> > insufficient community interest may prevent your EIP from moving forward, especially if it is a Core EIP.
> 
> Any way you'd suggest rephrasing that part?

Nope, I agree. I think that line is sufficient.

:+1:

Reply:
@carver @madeoftin I think my latest commit (removing `technically complex` and `simple`) addresses the comments in this thread. Feel-free to re-open if you feel otherwise :-)